### ISSUE TITLE
Detect dark mode on Windows 10

### ISSUE THREAD

**Issue by drinkitup777** [albertosottile/darkdetect#2]
Apparently Windows 10 has dark mode too. I wrote this if you would consider extending `darkdetect` functionality to Windows.

```python3
import platform

if platform.system() == 'Windows':
    import winreg


def is_windows_using_dark_theme():
    value = 1 # default to light theme

    try:
        with winreg.OpenKey(winreg.HKEY_CURRENT_USER, 'Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize') as key:
            value, value_type = winreg.QueryValueEx(key, 'AppsUseLightTheme')
    except OSError as e:
        pass
    
    return value > 0
```

**Comment by martinet101** [comment]
I've tried this and it works perfectly.

**Comment by albertosottile** [comment]
Closed in #4. To be released soon in 0.2.
